config-model/src/main/java/com/yahoo/searchdefinition/parser/ParsedAnnotation.java

```diff
@@ -9,13 +14,17 @@
 class ParsedAnnotation {
 
     private final String name;
+    private Optional<ParsedStruct> wrappedStruct;
```

Comment:
Intellij tells me that using Optional on parameters or member is bad practice.
It recomends using on return values only.